users/users/keycloak/utils.py

```diff
@@ -17,20 +18,20 @@ def create_filters(users: Users) -> Dict[str, Union[str, None]]:
         else {}
     )
 
+RETRIES = 2
 
 async def get_username(
-    access_token: schemas.ClientViewsUsersAccessToken, user_id: str
+    access_token: schemas.AdminAccessToken, user_id: str
 ):
-    for _ in range(2):
+    token = await access_token.token()
+    for _ in range(RETRIES):
         try:
-            return await query.get_user(realm, access_token.token, user_id)
-        except aiohttp.ClientResponseError as e:
-            if e.status in {401, 403}:
-                Logger.info(
-                    "Access Token for ClientViewsAllUsers_Role is expired. "
-                    "Creating new one"
-                )
+            return await query.get_user(realm, token, user_id)
+        except aiohttp.ClientResponseError as err:
+            Logger.exception(err)
```

Comment:
Same as previous comment
```suggestion
            logger.exception("Couldn't fetch username")
```